superduper/base/datalayer.py

```diff
@@ -260,6 +274,7 @@ def _post_query(self, table: str, ids: t.Sequence[str], type_: str):
             CFG.output_prefix
         ):
             logging.info(f'CDC for {table} is enabled')
+            assert self.cluster is not None
```

Comment:
Why do you need this one here ? If it is, it will throw an error on the exact next step.

Reply:
This makes sure the cluster part isn’t None, so the next step won’t throw an exception.